### ISSUE TITLE
make device list private / hide device list

### ISSUE THREAD

**Issue by awalland** [keybase/keybase-issues#3991]
i think there should be an option to make my device list private.

i think it's nobodys business which and how many devices i use.

**Comment by haukened** [comment]
The primary focus of keybase is establishing trust and identity, in order to have properly secured communications. If others can't see the status of your device, the trust model that keybase is established upon doesn't work.

**Comment by Rudi9719** [comment]
I'd also like to note that without publicising your device list/those device's keys, how would people encrypt data for those devices?

**Comment by awalland** [comment]
well then at least the name should be hidden and only the keys should be visible. most people will use descriptive names for their devices and it *is* valuable personal information if someone owns some notebook and a smartphone or 5 devices starting with a small i.

**Comment by Anynomouss** [comment]
I completely agree this information should not be made public. There is no problem having a list of devices, but descriptive names should be hashed and not provide any more detail than for example whether its a phone, PC etc. Having public devices names makes it rather easy to get IP and location and then address... It is not that I do not trust KeyBase with this information, but the whole internet can get this information from any KeyBase users, know which devices they own and then target any of them. I would go as far to say that this is not only a privacy problem but a security threat to all KeyBase users.
